docs/i18n/portuguese/how-to-work-on-coding-challenges.md

```diff
@@ -432,8 +435,8 @@ function myFunc() {
 
 #### Code Explanation
 
-- Code explanation goes here
-- Code explanation goes here
+- A explicação do código fica aqui
```

Comment:
I am not sure why this double line has the same message.

Reply:
This one could probably be fixed, yeah (in the English file via PR)